src/test/socket/accept/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
-# runs the shadow tests with only the passing tests, not all tests
 add_test(NAME accept COMMAND ../../target/debug/test_accept)
-add_test(NAME accept-shadow-ptrace COMMAND ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=ptrace -l debug -d accept.shadow-ptrace.data ${CMAKE_CURRENT_SOURCE_DIR}/accept.shadow.config.xml)
-add_test(NAME accept-shadow-preload COMMAND ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=preload -l debug -d accept.shadow-preload.data ${CMAKE_CURRENT_SOURCE_DIR}/accept.shadow.config.xml)
-set_property(TEST accept accept-shadow-ptrace accept-shadow-preload PROPERTY ENVIRONMENT "RUST_BACKTRACE=1")
+foreach(Method ptrace preload)
+    # runs the shadow tests with only the passing tests, not all tests
+    add_test(NAME accept-shadow-${Method} COMMAND ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=${Method}
+	    -l debug -d accept.shadow-${Method}.data ${CMAKE_CURRENT_SOURCE_DIR}/accept.shadow.config.xml)
+endforeach()
+
+set(TestNames accept accept-shadow-ptrace accept-shadow-preload)
```

Comment:
Same comments as your other PR:

* Consider appending to TestNames next to the corresponding add_test commands.
* Please briefly comment the FAIL_REGULAR_EXPRESSION bit, including a link to the relevant issue if applicable.

Reply:
Added these changes.